### DISCUSSION TITLE
Free Figma/Sketch Bootstrap 5 UI Kit

### DISCUSSION THREAD

**Discussion by ajaypatelaj** [twbs/bootstrap#34166] [Show & Tell]
Hi All! :wave:

We’re really excited to finally release Free Figma/Sketch Bootstrap 5, UI Kit, By [ThemeSelection](https://themeselection.com/)! :tada:

Design faster with a collection of prebuilt Figma/Sketch components with powerful auto/smart layout and resizing constraints.:rocket:

### :link:   Useful links:

**Demo:**
- Figma: https://www.figma.com/file/Cy8pNqIB9Px6zgR28lbhf9/Free-Figma-Bootstrap-5-UI-Kit?node-id=134%3A0
- Sketch: https://www.youtube.com/watch?v=CmnBm7yNE5A

**Download:**
- Figma: https://themeselection.com/products/free-figma-bootstrap-ui-kit/
- Sketch:https://themeselection.com/products/free-sketch-bootstrap-ui-kit/

#### Why Figma/Sketch Bootstrap UI Kit? :tada:
Figma/Sketch Bootstrap UI Kit comprising of 300+ organized Bootstrap 5 components built with atomic design system & auto layout. Kick start your next Figma project in no time.

#### Complete Figma Figma/Sketch Kit :sunglasses:
Figma/Sketch Bootstrap UI Kit is a comprehensive and easy-to-use Figma Library with organized components & atoms. It includes Bootstrap’s colors, grids, and typography so that you can easily customize it to fit your brand or product.

#### Design with Auto/Smart Layout :heart_eyes:
Auto/Smart Layout helps you to design your website or application faster and saves hours of work!

#### Effortless Resizing :scream:
Resize any components horizontally or vertically like a pro!

#### Design & Dev Consistency :male-technologist:
Layers and groups are made to match Bootstrap class names, with components following the exact order of the official Bootstrap documentation. it boosts your workflow and bridges the gap between designer & developer.

#### 300+ Organized Components 👨‍💻
Ready to use organized components with a consistent naming convention, that allows you to search and import easily.

#### Easy to customize :gear:
Bootstrap UI Kit helps you to easily set up colors, typography & border-radius that changes everywhere instantly in UI Kit. Also, allows you to easily update the state of the component.

Stay Safe :+1:

**Comment by monab** [comment]
Amazing Work...!!!!

**Comment by alokVishu** [comment]
Awesome work...!! i love it...
